src/App.css

```diff
@@ -147,3 +147,7 @@ li.panel .button-list {
 .scope__checkbox:checked ~ .scope__destinataire {
   max-height: 100px;
 }
+
+.side-pane__menu a {
+  text-decoration: none;
```

Comment:
do we know why we have to do this ?

Reply:
Because, in the template, the menu items are not `<a>` links, they are plain text.